docs/requirements.in

```diff
@@ -1,3 +1,5 @@
+emoji
 myst-parser
+pip-tools
```

Comment:
It might be a bad idea to let `pip-tools` manage itself.
I don't think it is a build dependency either?